README.md

```diff
@@ -16,7 +16,13 @@ If there is something you'd like to see Sourcegraph be able to do from the CLI,
 
 ## Installation
 
-**NOTE:** To get the best version for _your_ Sourcegraph instance, simply replace `sourcegraph.com` in the commands below with your own Sourcegraph URL and the latest version compatible with your instance will be provided. If you are running a Sourcegraph version older than 3.12, run the commands below instead.
+The preferred method of installation is to ask _your_ Sourcegraph instance for the latest compatible version. To do this, replace `https://sourcegraph.com` in the commands below with the address of your instance. If you are running a Sourcegraph version older than 3.12, these URLs will not be available and you must install directly from sourcegraph.com (by running the following commands _exactly_), or from one of the published [releases on GitHub](https://github.com/sourcegraph/src-cli/releases).
```

Comment:
While there's a bit more information here, I found the previous wording easier to follow. Specifically I think we've folded together the "preferred" and "exception" cases a little bit too much. I suggest we try separating them explicitly, for example:

~~~suggestion
**For Sourcegraph 3.12 and newer:** The preferred method of installation is to fetch the latest compatible version from _your_ Sourcegraph instance. To do this, replace `https://sourcegraph.com` with the address of _your instance_ in the commands below.

**For Sourcegraph prior to 3.12:** Sourcegraph versions older than 3.12 do not support these installation URLs, and you must install directly from sourcegraph.com by running the commands below _exactly_ as shown.  Alternatively, you may also install one of the published [releases on GitHub](https://github.com/sourcegraph/src-cli/releases).
~~~

Reply:
Updated.